playground/infrastructure/cloudbuild/cloudbuild_playground_infra.sh

```diff
@@ -40,6 +40,8 @@ git clone --branch $BRANCH_NAME $REPO_NAME --single-branch
 cd beam
 mkdir playground/terraform/environment/$ENVIRONMENT_NAME
  
+gcloud create bucket gs://$TF_VAR_state_bucket --location $TF_VAR_region --project $TF_VAR_project_id --pap
```

Comment:
Check return value and exit the script if bucket can't be created. We should consider the case when the name is not unique